apps/docs/.vitepress/config.ts

```diff
@@ -0,0 +1,311 @@
+import { defineConfig } from 'vitepress';
+import { codeInContextPlugin } from './plugins/codeInContextPlugin';
+import { snippetPlugin } from './plugins/snippetPlugin';
+
+export default defineConfig({
+  title: 'Fuels-ts',
+  description: 'FuelLabs Typescript SDK',
```

Comment:
```suggestion
  description: 'Fuel Labs Typescript SDK',
```

Sorry for the nitpicking 😅